INSTALL

```diff
@@ -714,6 +722,10 @@ The specific libraries and their options are:
   Options: RAJA_DIR, RAJA_OPT, RAJA_LIB.
   Versions: RAJA >= 0.13.0.
 
+- Caliper (optional), used when MFEM_USE_CALIPER = YES.
+  URL: https://github.com/LLNL/Caliper
+  Options: CALIPER_DIR
```

Comment:
It may be good to add a version requirement for Caliper, similar to other packages. If you have not tested with older versions, just use the version you tested with and add "(may work with older versions too)" -- no need to check which older versions actually work.

Reply:
It should work with older version, however, since it is tested with the current one 2.5.0 I have mentioned it in the INSTALL file